src/Sylius/Bundle/AdminBundle/MessageHandler/CreateAdminUserResult.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\MessageHandler;
+
+
+class CreateAdminUserResult
+{
+    public function __construct(private bool $hasViolations, private iterable $violationMessages = [])
```

Comment:
What's the point of setting `hasViolations` when, it'd assume, it's dependent on whether `violationMessages` is empty or not?